cfme/tests/webui/test_advanced_search.py

```diff
@@ -111,3 +135,86 @@ def test_can_open_advanced_search(params, appliance):
         pytest.fail("Advanced search cannot be opened for {}_{}".format(
             params[2], params[1].lower())
         )
+
+
+# @pytest.mark.uncollectif(lambda params: ((params[0] == TemplatesImages and BZ(1626579)) or
+#                                          (params[0] == MyService and BZ(1627078))))
+def test_filter_crud(params, appliance):
```

Comment:
**REQUIRED**
It's a mess, this function is too big, consider to divide it on some smaller functions.

Reply:
Could you help me how can I divide it in multiple smaller functions? Everything I tried broke the uncollectif.